neo/core/event.py

```diff
@@ -221,3 +221,25 @@ def as_quantity(self):
         Return the event times as a quantities array.
         """
         return self.view(pq.Quantity)
+
+    def to_epoch(self):
+        """
+        Transform Events into an array of Epoch 
+        """
+        from neo.core import Epoch, Event, Segment
+
+        print("self.time : " + str(self.times))
+        print("self.label : " + str(self.labels))
+        i = 0
+        duration = []
+        for t in self.times:
```

Comment:
variable t is never used. Maybe it would be better to use 'for i in range(len(self.times)):'?